apps/web/src/components/settings/RegistrationForm/RegisterFormSettings.tsx

```diff
@@ -117,17 +116,16 @@ export default function RegisterFormSettings({
 		}
 	}, [universityValue]);
 
+	let newResumeLink: string = c.noResumeProvidedURL;
 	async function onSubmit(
 		data: z.infer<typeof RegistrationSettingsFormValidator>,
 	) {
-		let resume: string = c.noResumeProvidedURL;
-
 		if (uploadedFile) {
 			const newBlob = await put(uploadedFile.name, uploadedFile, {
```

Comment:
The `put` command should prepend the `bucketResumeUploadUrl` from the config to ensure that all of the resumes from the respective hackathon stay there. Should be something like `'c.bucketResumeBaseUploadUrl/uploadedFile.name'`. You can also search for `fileLocation` which is a variable inside the `RegisterForm` component.